src/dao.cpp

```diff
@@ -3517,6 +3561,7 @@ void dao::reset() {
   delete_table<election_vote_table>(get_self(), 2);
   delete_table<election_vote_table>(get_self(), 3);
   delete_table<token_to_dao_table>(get_self(), get_self().value);
```

Comment:
should this be reward_token_to_dao_table instead of the old "token_to_dao_table"?